science/plumed/Portfile

```diff
@@ -0,0 +1,140 @@
+# sure that even if the user has that package already installed plumed is
+# not going to use it.
+configure.args-append \
+               --disable-libsearch \
+               --disable-molfile-plugins \
+               --disable-mpi 
+
+# Hardcode path for libplumedKernel.dylib.
+# This allows to patch MD codes using the --runtime option but using as
+# default kernel the installed one. In this way, MacPorts users
+# can just use patched MD codes with the installed plumed or replace it
+# by setting PLUMED_KERNEL at runtime
+configure.cppflags-append "-D__PLUMED_DEFAULT_KERNEL=${prefix}/lib/libplumedKernel.dylib"
+
+compilers.choose    cc cxx
+mpi.setup
+
+if {[mpi_variant_isset]} {
+  configure.args-replace --disable-mpi --enable-mpi
+}
+
+variant matheval description {Enable libmatheval} {
+  configure.ldflags-append -lmatheval
+  depends_lib-append port:libmatheval
+}
+
+variant xdrfile description {Enable xdrfile} {
+  configure.ldflags-append -lxdrfile
+  depends_lib-append port:xdrfile
+}
+
+variant zlib description {Enable zlib} {
+  configure.ldflags-append -lz
+  depends_lib-append port:zlib
+}
+
+variant gsl description {Enable gsl} {
+  configure.ldflags-append -lgsl
+  depends_lib-append port:gsl
+}
+
+variant molfile description {Enable molfile - includes some BSD code} {
+  license ${license} BSD
+  configure.args-replace --disable-molfile-plugins --enable-molfile-plugins
+}
+
+variant doc description {Also compile documentation} {
+  distfiles-append plumed-doc-${version}.tgz
+  depends_build-append port:doxygen
+  depends_build-append port:graphviz
+  build.target all_plus_docs
+}
+
+# this is a tcl variable contains a concatenated list of modules
+# to be activated
+
+set plumed_modules reset
+
+# this will automatically create variants for each optional module
+variant mod_crystallization description {Enable crystallization module} {
+  set plumed_modules ${plumed_modules}+crystallization
+}
+
+variant mod_manyrestraints description {Enable manyrestraints module} {
+  set plumed_modules ${plumed_modules}+manyrestraints
+}
+
+variant mod_adjmat description {Enable adjmat module} {
+  set plumed_modules ${plumed_modules}+adjmat
+}
+
+variant allmodules description {Enable all optional modules} {
+  set plumed_modules ${plumed_modules}+crystallization
+  set plumed_modules ${plumed_modules}+manyrestraints
+  set plumed_modules ${plumed_modules}+adjmat
+}
+
+
+pre-configure {
+  # commands should be included in a pre-configure block to access tcl variables
+  configure.args-append --enable-modules=${plumed_modules}
+  configure.ldflags-append ${linalglib}
+}
+
+# Default variants include no optional modules.
+# Additionally, the link libraries that are very useful (e.g. matheval of xdrfile)
+# and quick to compile
+default_variants +matheval +xdrfile +zlib +molfile
+
+# This is required since PLUMED installation does not do it explicitly.
+# It might be removed when this will be incorporated upstream
```

Comment:
Yes, fix this upstream please. You may temporary do this, but this should really be fixed. (Don't autotools automatically take care of this?)